docs/sources/flow/reference/components/prometheus.scrape.md

```diff
@@ -54,38 +54,29 @@ Name | Type | Description | Default | Required
 `label_limit`              | `uint`     | More than this many labels post metric-relabeling causes the scrape to fail. | | no
 `label_name_length_limit`  | `uint`     | More than this label name length post metric-relabeling causes the scrape to fail. | | no
 `label_value_length_limit` | `uint`     | More than this label value length post metric-relabeling causes the scrape to fail. | | no
+{{< docs/shared lookup="flow/reference/components/http-client-config-block.md" source="agent" >}}
```

Comment:
This included  block is messing up how the preceding table is rendering.  Just the names are showing and not columns 2 - 4. I think you need a blank line and also the "authorization block" header as in: https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.remote_write/#authorization-block

Reply:
I tried a new line, but I can't personally see if the table will render with the additional rows as intended. In this case the http_client_config is no longer a block section/table on its own but is now additional rows of the Arguments table.

Reply:
You run `make docs` to see how it will render.  It looks like the included file has a header row so that would prevent it from rendering the way you want.

Reply:
Excellent, thank you for the pointer on running locally. This should be all set for now. We are looking into a future way to put the table data in a shared file.